Assets/Scripts/ColorRandomizer.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ColorRandomizer : MonoBehaviour
+{
+    void Awake()
+    {
+        Color green = new(0.4334f, 0.78f, 0.234f);
```

Comment:
Celá táto časť sa dá zjednodušiť takýmto štýlom:

```
Color[] colors = new Color[] {
new Color()...
}

int randomIndex = Random.Range(0, colors.Length);

gameObject.GetComponent<SpriteRenderer>().color = colors[randomIndex];
```

Reply:
Chapem...

Reply:
Ale, naschvál som to rozpisoval aby bolo vidno ktorá farba je ktorá a keď tie farby bude zasa treba, tak si ich bude môcť hocikto zasa skopírovať z tade a nebude treba to zisťovať